addons/point_of_sale/static/src/app/utils/money_details_popup/money_details_popup.js

```diff
@@ -21,6 +21,11 @@ export class MoneyDetailsPopup extends AbstractAwaitablePopup {
         });
     }
     computeTotal(moneyDetails = this.state.moneyDetails) {
+        Object.keys(moneyDetails).forEach((key) => {
+            if (isNaN(moneyDetails[key])) {
+                moneyDetails[key] = 0;
+            }
+        });
```

Comment:
```suggestion
```
Good job finding the place. But mutating the state in a getter is not a good practice. Getters should not have side effects. I'd suggest you change the return value into:

```js
        return Object.entries(moneyDetails).reduce((total, money) => {
            const [valueStr, inputQty] = money;
            const value = parseFloat(valueStr);
            const quantity = isNaN(inputQty) ? 0 : inputQty;
            return total + value * quantity;
        }, 0);
```